applications/task-manager/main.cpp

```diff
@@ -22,19 +22,30 @@ struct TaskManagerWindow
     TaskModel *table_model;
     Timer *table_timer;
 };
+TaskManagerWindow *window;
 
 void widget_ram_update(Graph *widget)
 {
     SystemStatus status = system_get_status();
 
     graph_record(widget, status.used_ram / (double)status.total_ram);
+    
+    char buffer[200];
+    int percentage = (int)100 * status.used_ram / status.total_ram;
```

Comment:
I don't know why but the reported ram usage was 0% on your screenshot :thinking:

Reply:
The percentage changes a lot. It can on 10% and jump down to 0%